payment/src/main/java/org/killbill/billing/payment/invoice/InvoicePaymentControlPluginApi.java

```diff
@@ -642,6 +652,7 @@ public boolean apply(final PaymentTransactionModelDao input) {
                                                           successfulTransaction.getCurrency(),
                                                           successfulTransaction.getProcessedCurrency(),
                                                           successfulTransaction.getPaymentId(),
+                                                          incompleteInvoicePayment.getId(),
```

Comment:
Is that correct?

Reply:
Good catch - fixed in https://github.com/killbill/killbill/pull/1274/commits/c9a71e60bb54318b67cb4cfb570f80592240977f. My test is just checking that value is not null, but i did verify this time this is correct.